packages/x-components/src/x-modules/extra-params/components/snippet-config-extra-params.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <ExtraParams :values="extraRequestParams" />
+</template>
+
+<script lang="ts">
+  import Vue from 'vue';
+  import { Component, Watch } from 'vue-property-decorator';
+  import { xComponentMixin, XInject } from '../../../components';
+  import { Dictionary, forEach } from '../../../utils';
+  import { SnippetConfig } from '../../../x-installer';
+  import { extraParamsXModule } from '../x-module';
+  import ExtraParams from './extra-params.vue';
+
+  /**
+   * It renders a {@link ExtraParams} component and pass the snippetConfig.
```

Comment:
You don't have to describe how this component does things, you have to describe what it does. Maybe something in the line of this?

> Extracts the extra parameters from the {@link SnippetConfig} and syncs it with the request objects of every x-module.

And I'm missing class properties/methods doc, and the `<docs>` block